plugin/db/util/driverutil.go

```diff
@@ -28,12 +28,20 @@ func FormatErrorWithQuery(err error, query string) error {
 
 // ApplyMultiStatements will apply the split statements from scanner.
 func ApplyMultiStatements(sc io.Reader, f func(string) error) error {
-	scanner := bufio.NewScanner(sc)
+	reader := bufio.NewReader(sc)
```

Comment:
Is it possible to merge the usage of this and the parser tokenizer?

Reply:
If we want to merge them, we need a common SQL-splitter in the tokenizer. But we only have the SQL-splitter for MySQL and PostgreSQL.
I can do this in the following PR but it's not a good quick-fix.

Reply:
SGTM. Then let's duplicate the randomString generation for now and remove one of the two later because only one of code path needs a test on this.

Reply:
Added a "TODO(x): xxx." perhaps.

Reply:
Done.